locale/en_US/common.po

```diff
@@ -1990,3 +1990,9 @@ msgstr "Trying to run a transformation that is not supported in your installatio
 
 msgid "filter.import.error.couldNotImportNode"
 msgstr "Could not import XML Node"
+
+msgid "navigation.tools.jobs"
```

Comment:
These are only viewable by the manager, right? I'd suggest keeping them out of `common.po` if that's the case. The `manager.po` and `admin.po` files are more appropriate.